src/routes/citations/handlers.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable import/prefer-default-export */
+import httpCodes from 'http-codes';
+import {
+  DECIMAL_BASE,
+} from '@ttahub/common';
+import handleErrors from '../../lib/apiErrorHandler';
+import User from '../../policies/user';
+import { currentUserId } from '../../services/currentUser';
+import { userById } from '../../services/users';
+import { getCitationsByGrantIds } from '../../services/citations';
+
+const namespace = 'SERVICE:CITATIONS';
+
+const logContext = { namespace };
+
+export const getCitationsByGrants = async (req, res) => {
+  try {
+    // Get the grant we need citations for.
+    const { regionId, reportStartDate } = req.params;
```

Comment:
I think you want reportStartDate in the query, not the params

Reply:
I have updated this in the PR that calls the BE. Region ID is the only one in the params. Both regionId and start date are in the query.